kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -198,4 +199,21 @@ public SubscriptionPack getSubscriptionPack(String name) {
         return subscriptionPackDataService.byName(name);
     }
 
+
+    /**
+     * To be used by ITs only!
+     */
+    public void deleteAll() {
```

Comment:
didn't realize that we were going to force ITs to use the service too (instead of data layer)...but it makes sense